scripts/zowe-install-MVS.sh

```diff
@@ -133,6 +133,22 @@ else
     echo "  $script failed to create ${ZOWE_DSN_PREFIX}.SZWESAMP, RC=$rc" >> $LOG_FILE
 fi
 
+# 3. {datasetprefix}.SZWEPLUG
+
+tsocmd "delete '${ZOWE_DSN_PREFIX}.SZWESAMP' " >> ${LOG_FILE} 2>&1
+
+# TODO replace by allocate-dataset.sh call to resuse VOLSER support
+tsocmd "allocate new da('${ZOWE_DSN_PREFIX}.SZWEPLUG') " \
+    "dsntype(library) dsorg(po) recfm(u) lrecl(0) bLKSIZE(32760)" \
```

Comment:
that seems correct, to basically have it be identical in attribute to SZWEAUTH. do you have a test that shows this has been successfully created?